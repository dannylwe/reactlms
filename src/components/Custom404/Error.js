import React, { Component } from "react";
import { Link } from "react-router-dom";

class Error extends Component {
  render() {
    return (
      <div>
        <h3>OOPS! That URL does not exist</h3>
        <Link to="/">Go back to home</Link>
      </div>
    );
  }
}

export default Error;
