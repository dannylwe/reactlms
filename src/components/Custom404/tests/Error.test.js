import React from 'react';
import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router-dom';
import Error from '../Error';

describe('Error page', () => {
  it('should render without failing', () => {
    const wrapper = renderer.create(<MemoryRouter><Error /></MemoryRouter>);
    expect(wrapper.toJSON()).toMatchSnapshot();
  });
});
