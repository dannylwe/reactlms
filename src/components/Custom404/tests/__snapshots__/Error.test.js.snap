// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error page should render without failing 1`] = `
<div
  className="error-heading"
>
  <h3>
    OOPS! That URL does not exist
  </h3>
  <a
    href="/"
    onClick={[Function]}
  >
    Go back to home
  </a>
</div>
`;
